### DISCUSSION TITLE
BFE是否支持租户(Product)级别的tls(https)证书配置？

### DISCUSSION THREAD

**Discussion by zytwhut** [bfenetworks/bfe#1142] [General]
1. 目前BFE开源版本是否支持租户(Product)级别的tls(https)证书配置，即为  不同的租户(Product)配置不同的tls(https)证书？
2. 如果不支持，后续是否有支持计划？
3. 如果支持，应该如何配置，是否有相关说明文档？其对应的代码逻辑在哪个地方？如果支持的话，感觉逻辑应该是  (1)根据Host找到Product->(2)根据product找到tls证书->(3)tls握手->后续操作？

**Comment by mileszhang2016** [comment]
1. 目前BFE开源版本不支持租户级别的证书配置
2. 我们确实收到了部分用户提到这方面的需求，但是还没有排期开发
3. 针对租户来选择证书，可能有两种场景：
（1）基于vip来选择
（2）在SNI场景下基于域名来选择
